drivers/spi/Kconfig.xec_qmspi_v2

```diff
@@ -0,0 +1,12 @@
+# Microchip XEC QMSPI V2 supporting MEC172x
+
+# Copyright (c) 2022 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config SPI_XEC_QMSPI_V2
+	bool "Microchip XEC QMSPI V2 driver"
+	default n
```

Comment:
`default n` is the default for booleans. No need to write it out.

Reply:
using choice now